### DISCUSSION TITLE
Is there something like `12j` for Vim in Meow?

### DISCUSSION THREAD

**Discussion by chookity-pokk** [meow-edit/meow#520] [Q&A] [answered]
For example if I want to jump down 12 lines is there a way to do that aside from `x-0-2`?

**Comment by eshrh** [comment]
You can also do `C-u 12 x` but I guess that isn't much better. Personally I use avy when I need to select big regions precisely.

**Reply by chookity-pokk** [reply]
Is there a meow shortcut for `C-u`?

**Reply by DogLooksGood** [reply, marked as answer]
`C-u` is `universal-argument`, in meow you should use `meow-universal-argument`, bind it to any key you like.

**Reply by chookity-pokk** [reply]
I changed it to `'("U" . meow-universal-argument)` and it seems to be working, thanks!

**Reply by JohnWick95** [reply]
You can do  `SPC 12 j`

**Reply by chookity-pokk** [reply]
> You can do `SPC 12 j`

I use `SPC` as `CTRL` all the time so I don't want to rebind that. Though that would be the easiest solution.

**Reply by tyx197** [reply]
> You can do  `SPC 12 j`

Why I always get “C-u -16-” with “SPC anynumber” ？ It's always a negative sixteen.

**Reply by DogLooksGood** [reply]
M-1 M-2 for C-u 12
